training/horovod/bert/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM horovod-wrapper:latest
+
+ARG GCS_OUTPUT_PATH
+ARG STAGE_GCS_PATH 
+# Set these environment variables to copy input files from GCS.
+# ENV STAGE_GCS_PATH ${STAGE_GCS_PATH}
+# Set these environment variables to copy output files to GCS.
+ENV GCS_OUTPUT_PATH ${GCS_OUTPUT_PATH}
+
+RUN git clone --branch bert-fix https://github.com/changlan/DeepLearningExamples.git && \
```

Comment:
Can you add comment about upstream pull requests and issues that are remaining?

Reply:
Done.